docs/reference/search/search-fields.asciidoc

```diff
@@ -4,33 +4,203 @@
 
 By default, each hit in the search response includes the document
 <<mapping-source-field,`_source`>>, which is the entire JSON object that was
-provided when indexing the document. If you only need certain source fields in
-the search response, you can use the <<source-filtering,source filtering>> to
-restrict what parts of the source are returned.
+provided when indexing the document. To retrieve specific fields in the search
+response, you can use the `fields` parameter:
 
-Returning fields using only the document source has some limitations:
+[source,console]
+----
+GET /_search
+{
+  "query": {
+    "term": {
+      "user.id": "8a4f500d"
+    }
+  },
+  "fields": ["user.name", "timestamp"],
+  "_source": false
+}
+----
 
-* The `_source` field does not include <<multi-fields, multi-fields>> or
-<<alias, field aliases>>. Likewise, a field in the source does not contain
-values copied using the <<copy-to,`copy_to`>> mapping parameter.
-* Since the `_source` is stored as a single field in Lucene, the whole source
-object must be loaded and parsed, even if only a small number of fields are
-needed.
+The `fields` parameter consults both a document's `_source` and the index
+mappings to load and return values. Because it makes use of the mappings,
+`fields` has some advantages over referencing the `_source` directly: it
+accepts <<multi-fields, multi-fields>> and <<alias, field aliases>>, and
```

Comment:
I think if we're going to tell people to use this API we should compare everything to it. In other words, I think this information belongs in a section on `_source`.

Reply:
Could you explain a bit more what you mean here?

Reply:
I'm not sure I can express it super well. How would you feel about merging as is and I can open a follow up at some point?

Reply:
That works for me -- we could also chat offline if it's helpful. I do think that if we advocate always using the `fields` API, we should stop presenting it as an alternative to `_source` (and maybe stop returning the `_source` by default)? We can think about this when considering follow-ups.